tests/test_models.py

```diff
@@ -89,3 +94,110 @@ def test_string_conversion(self, method):
             user_id=user_id
         )
         self.assertEqual(method(customer_user), expected_to_str)
+
+
+@mark.django_db
+@ddt.ddt
+class TestEnterpriseCustomerBrandingConfiguration(unittest.TestCase):
+    """
+    Tests of the EnterpriseCustomerBrandingConfiguration model.
+    """
+
+    def get_enterprise_customer(self):
+        """
+        @:return EnterpriseCustomer object.
```

Comment:
@asadiqbal08 why `@:`? I'm not an expert, but I can't remember any restructured text directive like that.